lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -64,6 +98,11 @@ export default {
 				return Object.values(BADGE_COLORS).includes(value);
 			},
 		},
+		icon: {
+			type: String,
+			default: ICONS.FA_SHARE_NODES,
```

Comment:
hmm, did UX told something about setting it as default value in DS? For me `null` is more flexible default.

Reply:
Yes, it should be null - https://miro.com/app/board/o9J_knPnexA=/?moveToWidget=3458764569145573275&cot=14